QuizApp/Client/Account/AccountNetworkClient.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+
+class AccountNetworkClient: AccountNetworkClientProtocol {
+
+    private let baseNetworkClient: BaseNetworkClientProtocol
+
+    init(baseNetworkClient: BaseNetworkClientProtocol) {
+        self.baseNetworkClient = baseNetworkClient
+    }
+
+    func getAccountDetails() async throws -> AccountDetailsResponse {
```

Comment:
Functions that return a value and take no arguments should be computed properties.